Remove files related to stream device and format remaining

mac-kan opened the pull request:
Remove .template, .proto and .substitutions files using stream device.
Remove calls to stream device from the startup files.
Format the remaining .template file.
Rename Boy folder to opi.

mac-kan committed:
Rename Boy folder to opi

mac-kan committed:
adsExApp: Remove stream device example

Remove remnants of stream device db:s from the example app.

mac-kan committed:
adsExApp: Use dbformat

Use dbformat (https://gitlab.esss.lu.se/e3/dbformat) to format the
remaining db file in adsExApp.

mac-kan committed:
Remove stream device from startup scripts

Remove stream device related calls from the startup example scripts.

mac-kan committed:
adsApp: Remove files using stream device

Remove .template, .proto and .substitutions example files using
stream device.

simon-ess commented:
LGTM

simon-ess commented:
Module review comment: All e3/ESS-specific artifacts (e.g. `require ads`) should be removed from the startup scripts.

mac-kan committed:
Remove require from startup scripts

Remove the references to require from the startup scripts since this
is a community module.
